opm/simulators/flow/Transmissibility_impl.hpp

```diff
@@ -23,21 +23,27 @@
 #ifndef OPM_TRANSMISSIBILITY_IMPL_HPP
 #define OPM_TRANSMISSIBILITY_IMPL_HPP
 
+#ifndef OPM_TRANSMISSIBILITY_HPP
+#include <config.h>
```

Comment:
:thinking: Isn't this a header and config.h for *.cpp files?

Reply:
see https://github.com/OPM/opm-simulators/pull/5835 (and it's parent) for explanation.

Reply:
Please add a short comment anyway.
Not everybody remembers all OPM PRs and files

Reply:
I second that, I could not find the relevant discussion?

Reply:
comment added.